ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Public/CookiePreferences.cshtml

```diff
@@ -28,16 +28,19 @@
         <p class="govuk-body">
             Cookies are small files saved on your phone, tablet or computer when you visit a website.
         </p>
-        <p class="govuk-body">
-            We use cookies to make this site work and to collect information about how you use our service.
-        </p>
+       <p class="govuk-body">
```

Comment:
The indentation is inconsistent, is this just weirdness in the diffs being shown?